yarn-project/scripts/Earthfile

```diff
@@ -66,4 +66,29 @@ bench-comment:
     [ -f $BENCH_FOLDER/benchmark.json ] \
       && (cd /usr/src/yarn-project/scripts && AZTEC_BOT_COMMENTER_GITHUB_TOKEN=$AZTEC_BOT_COMMENTER_GITHUB_TOKEN yarn bench-comment) \
       || echo "No benchmark file found in $BENCH_FOLDER"
-  
\ No newline at end of file
+
+base-log-uploader:
+  # Install awscli on a fresh ubuntu, and copy the repo "scripts" folder, which we'll use to upload logs
+  # Note that we cannot do this LOCALLY because Earthly does not support using secrets locally
+  FROM ubuntu:noble
+  RUN apt update && \
+      apt install -y curl git jq unzip
+  RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip" && \
+      unzip awscliv2.zip && \
+      ./aws/install --bin-dir /usr/local/bin --install-dir /usr/local/aws-cli --update && \
+      rm -rf aws awscliv2.zip
+  COPY ../../+scripts/scripts /usr/src/scripts
+
+UPLOAD_LOGS:
+    FUNCTION
+    ARG PULL_REQUEST
+    ARG BRANCH
+    ARG COMMIT_HASH
+    ARG LOG_FILE=./log
+    LOCALLY
+    LET COMMIT_HASH="${COMMIT_HASH:-$(git rev-parse HEAD)}"
+    FROM +base-log-uploader
+    ENV PULL_REQUEST=$PULL_REQUEST
+    ENV BRANCH=$BRANCH
+    ENV COMMIT_HASH=$COMMIT_HASH
+    RUN --secret AWS_ACCESS_KEY_ID --secret AWS_SECRET_ACCESS_KEY /usr/src/scripts/logs/upload_logs_to_s3.sh $LOG_FILE
```

Comment:
We're missing the COPYing of the LOG_FILE into the uploader, right?

Reply:
Fixed in #6424